README.rst

```diff
@@ -47,6 +47,8 @@ You might also want to explore the alpha release of the
 `OpenFermion Cloud Library <https://github.com/quantumlib/OpenFermion/tree/master/cloud_library>`__
 where users can share and download precomputed molecular benchmark files.
 
+Check out other `projects and papers using OpenFermion <docs/other_projects.md>`__ for inspiration, and let us know if you've
```

Comment:
80 Characters/ line?

Reply:
I didn't see it too strict in other markdown files we've been using but I can update these lines to make sure they are